packages/caver-utils/src/index.d.ts

```diff
@@ -0,0 +1,203 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import BigNumber from 'bignumber.js'
+import BN from 'bn.js'
+import _ from 'lodash'
+import SignatureData from '../../caver-wallet/src/keyring/signatureData'
+import BaseUtils from './utils'
+import Iban from '../iban'
+
+export type Unit = 'peb' | 'kpeb' | 'Mpeb' | 'Gpeb' | 'Ston' | 'ston' | 'uKLAY' | 'mKLAY' | 'KLAY' | 'kKLAY' | 'MKLAY' | 'GKLAY' | 'TKLAY'
+
+export interface KlayUnit {
```

Comment:
This is also duplicate ;-;

Reply:
I abstracted to key and values. please check c46afc4

type `Unit` and interface `UnitMap` need also abstract?